openproblems/tasks/_batch_integration/batch_integration_embed/README.md

```diff
@@ -0,0 +1,54 @@
+<!--- TODO: add links --->
+
+# Batch integration embedding
+
+This is a sub-task of the overall batch integration task. Batch (or data) integration
+integrates datasets across batches that arise from various biological and technical
+sources. Methods that integrate batches typically have three different types of output:
+a corrected feature matrix, a joint embedding across batches, and/or an integrated
+cell-cell similarity graph (e.g., a kNN graph). This sub-task focuses on all methods
+that can output joint embeddings, and includes methods that canonically output corrected
+feature matrices with subsequent postprocessing to generate a joint embedding. Other
+sub-tasks for batch integration can be found for:
+
+* [graphs](../batch_integration_graph/), and
+* [corrected features](../batch_integration_features)
+
+This sub-task was taken from a
+[benchmarking study of data integration methods]
+(<https://www.biorxiv.org/content/10.1101/2020.05.22.111161v2>).
+
+## API
+
+Datasets should contain the following attributes:
+
+* `adata.obs["batch"]` with the batch covariate, and
+* `adata.obs["label"]` with the cell identity label
+* `adata.obsm['X_uni']` with a pre-integration embedding (usually PCA)
```

Comment:
I wonder if it's important to say always PCA or not. What do you think?

Reply:
as discussed during the meeting, I think it makes sense to require PCA